src/components/Dashboard/Dashboard.jsx

```diff
@@ -41,8 +41,8 @@ export class Dashboard extends React.Component {
                             <Timothy />
                         </div>
                         <div className={styles.centerBox}>
-                            {this.props.status !== statusTypes.GAME_OVER && this.props.playedCards.length > 0 && <PlayedCards />}
-                            {this.props.playCard && <MyCards />}
+                        {this.props.status !== statusTypes.WAITING_FOR_OTHER_PLAYERS && this.props.playedCards.length > 0 && <PlayedCards />}
```

Comment:
Won't this just show the played cards once everyone has played? I thought we wanted them to show with the backs on after a player has played their card